### PR TITLE
1st homework

### PR DESCRIPTION
**Pull request by Makarov-Ivan**
1st homework without using JSX.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+class MyComponent extends React.Component {
+    constructor() {
+        super()
+    };
+    render = () => getElement("div", null, "React.Component")
+}
+const myComponent = new MyComponent();
+
+class MyPureComponent extends React.PureComponent {
```
Comment (lyolya95) [inline index.js:16]: I think you need to divide this file at every component and give the right name.
For example
PureComponent.js
const PureComponent...
etc.

and files import to the index file

File: index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    constructor() {
+        super()
+    };
+    render = () => getElement('p', null, "i'm pure component")
+}
+const pureComponent = new MyPureComponent();
+
+const fnComponent = () => getElement('div', null, 'im functional component')
```
Comment (lyolya95) [inline index.js:24]: "I'm" please, be attentive

File: index.js (hunk 1 above)
Comment (Makarov-Ivan) [reply index.js]: Hi @lyolya95 , i changed something. So now there is a folder for this components

File: index.js (hunk 2 above)
Comment (Makarov-Ivan) [reply index.js]: yeah 😀

**Review by lyolya95 — APPROVED** [review]

**Pull request merged**